src/components/ErrorBoundary.jsx

```diff
@@ -6,6 +6,7 @@ const reportingURL = "https://github.com/unchained-capital/caravan/issues";
 
 class ErrorBoundary extends React.Component {
   static propTypes = {
+    // FIXME: should this really have two types?
```

Comment:
Not to be fixed, this really has to be one of two I think just because of the way React works. 
It could look like this:

```
<ErrorBoundary>
  <h1>Hello World</h1>
</ErrorBoundary>
```
or this:
```
<ErrorBoundary>
  <h1>Hello World</h1>
  <p>I have just made this an array of nodes</p>
</ErrorBoundary>
```

Technically we are using this around the whole app, and only once, so we can guess that it will only be used in one way or another, but it could be used in other places and this is a typical pattern for children nodes in a component.

Reply:
Okay, will remove the`// FIXME`s

Reply:
Updated.